utility-belt/pom.xml

```diff
@@ -0,0 +1,232 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--~
+  ~ Copyright 2017 Confluent Inc.
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~ http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  ~-->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>io.confluent</groupId>
+        <artifactId>common-docker</artifactId>
+        <version>4.1.2-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>utility-belt</artifactId>
+
+    <description>Utility classes for Confluent's Docker images</description>
+    <name>${project.artifactId}</name>
+
+    <packaging>jar</packaging>
+
+    <properties>
+        <assertj.version>2.2.0</assertj.version>
+        <apacheda.version>1.0.0-M33</apacheda.version>
+        <apacheds.version>2.0.0-M21</apacheds.version>
+        <argparse4j.version>0.7.0</argparse4j.version>
+        <bcpkix.version>1.54</bcpkix.version>
+        <gson.version>2.7</gson.version>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.apache.kafka</groupId>
```

Comment:
Not related to the selected line but a question for this PR. In the PR's description, you mentioned that this is to allow common to be built without Kafka. What's the reason that we need to do that / can you add more color to the description? I am sure that you have good reasons to do all this work, so just wanted to understand it. Thx.

Reply:
Because it's a heavyweight dependency, and doesn't need to be there? I thought this much was obvious to anyone who's ever tried to build common.

This dependency wasn't there until we merged the docker stuff, and it's a problem for packaging for reasons we don't need to get into in an OSS PR.

Reply:
I agree it's heavyweight, so if there is a better way to get what we want without it, sure. 

If the other problems are significant, I think it is fine to discuss in an OSS PR as we do want to be transparent as long as it doesn't divulge proprietary info that shouldn't be in the public. With that said, ok with making common lighter alone.